tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -73,8 +72,7 @@ virtual void TearDown() {
 
 INSTANTIATE_TEST_CASE_P(CTFont,
                         FontCopyName,
-                        ::testing::Values(::testing::make_tuple(kCTFontCopyrightNameKey, c_copyrightName),
```

Comment:
I'm not sure how difficult it would be to add the metadata, as that would make this much cleaner.  @DHowett-MSFT ?